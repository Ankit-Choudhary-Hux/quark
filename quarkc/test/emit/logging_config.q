void main(List<String> args) {
    logging.defaultConfig().setAppender(logging.stdout()).setLevel("DEBUG").configure();
    Logger l = new Logger("hello");
    l.trace("Should not see trace log");
    l.debug("Debug log");
    l.info("Info log");
    l.error("Error log");
}
