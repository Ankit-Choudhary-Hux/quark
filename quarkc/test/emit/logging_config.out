DEBUG quark.hello Debug log
INFO quark.hello Info log
ERROR quark.hello Error log
